### PR TITLE
Documentation: Add an example of a powershell script that uses the API

### LINKED ISSUE #921: Documentation: Add Downloading Using The API via Powershell
Originally requested in : https://adoptium.slack.com/archives/CFUJV9XV0/p1707878617367689

Add a powershell example to the api cookbook document.

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #921 

As provided by the community, I have added an example PowerShell script that utilises the API to download the JDKs.

# Checklist

- [X] You changed or added to the documentation

### CODE REVIEW

File: docs/cookbook.adoc [hunk 1]
```diff
@@ -145,6 +145,66 @@ The full set of filters and possible argument values is available in
 https://api.adoptium.net/q/swagger-ui/[the API documentation]
 and can be used to specify exactly the information you require.
 
+[#example-five]
+== Example Five: Download JDKs via the API using Powershell.
+
+Below is a sample script written in Powershell for Windows This script downloads and verifies the JDKs via the API, and performs checksum validation and notifications via email for the results. This script is provided as an example to demonstrate how the API can be used to automate tasks such as downloading and verifying the JDK.
```
Comment (johnoliver) [inline docs/cookbook.adoc:151]: Does it do email?

File: docs/cookbook.adoc [hunk 2]
```diff
@@ -145,6 +145,66 @@ The full set of filters and possible argument values is available in
...
+    $Platforms | ForEach-Object {
+        $Platform = $_
+        $Types = @('jdk','jre')
+        $Types | ForEach-Object {
+            $Type = $_
+            $ReleaseInfo = Invoke-WebRequest -Uri "https://api.adoptium.net/v3/assets/latest/$Release/hotspot?architecture=x64&image_type=$Type&os=$Platform&vendor=eclipse" -UseBasicParsing | ConvertFrom-Json
+            $CurrentDate = Get-Date -UFormat "%Y-%m-%dT%H:%M:%SZ"
+            $TimeDifference = New-TimeSpan -Start $ReleaseInfo.binary.updated_at -End $CurrentDate
+            
+                if ("$Platform" -eq "linux") {
```
Comment (johnoliver) [inline docs/cookbook.adoc:172]: the linux and windows paths look pretty similar, can probably be extracted to common code

File: docs/cookbook.adoc (hunk 1 above)
Comment (steelhead31) [reply docs/cookbook.adoc]: Not any longer :)

File: docs/cookbook.adoc (hunk 2 above)
Comment (steelhead31) [reply docs/cookbook.adoc:171]: Done, rationalised to a single path.

Comment (johnoliver) [reply docs/cookbook.adoc:171]: I think the if and else are now identical, but both paths still exist, can they be brought down to 1?

Comment (steelhead31) [reply docs/cookbook.adoc]: Yup, should be better now. Thanks!

**Review by johnoliver — APPROVED** [review]

**Pull request merged**